src/containers/org/DataSetPermissionsCard.js

```diff
@@ -47,7 +47,6 @@ const ORDERED_PERMISSIONS = [
   PermissionTypes.OWNER,
   PermissionTypes.READ,
   PermissionTypes.WRITE,
-  PermissionTypes.DISCOVER,
```

Comment:
what happened to this?

Reply:
https://jira.openlattice.com/browse/APPS-2381